pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -133,16 +133,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
+message PodSecurityContext {
+    // The configurations for the sandbox's namespaces.
+    // This will be used only if the PodSandbox uses namespace for isolation.
+    optional NamespaceOption namespace_options = 2;
+    // Optional SELinux context to be applied. If set in both SecurityContext
+    // and PodSecurityContext, the value specified in SecurityContext takes
+    // precedence for the container.
```

Comment:
Should kubelet or the runtime decide enforce this? I feel like kubelet should supply this in the `ContainerConfig` to make things simpler.